### ISSUE TITLE
How to select  the best result?

### ISSUE THREAD

**Issue by fake-warrior8** [haojc/ShufflingVideosForTSG#7]
Hi, I would like to ask if the best result is chosen based on the valid set or the test set? Is it use the mIoU or other metrics (like the mean of R=1, Iou=0.3, 0.5, 0.7)? How to reproduce the total results reported in the paper?

**Comment by haojc** [comment]
For Q1&2: We conduct hyperparameters search on the validation sets for all dataset splits. The best training epoch is chosen based on the validation set according to mIoU. 
For Q3: We have provided the pre-trained models (can be found in our **Features and Pretrained Models**) and the performance of these models is even slightly better than the results in the paper. The results in the paper are obtained by averaging the scores of three time runs. And you can also train models from scratch without any change of hyperparameters.

**Comment by fake-warrior8** [comment]
> For Q1&2: We conduct hyperparameters search on the validation sets for all dataset splits. The best training epoch is chosen based on the validation set according to mIoU. For Q3: We have provided the pre-trained models (can be found in our **Features and Pretrained Models**) and the performance of these models is even slightly better than the results in the paper. The results in the paper are obtained by averaging the scores of three time runs. And you can also train models from scratch without any change of hyperparameters.

Thank you for your reply!
